From: "Eduardo Lima (Etrunko)" <eduardo.lima@intel.com>
Date: Wed, 5 Feb 2014 15:05:49 -0200
Subject: Enable building multiple binary packages

With this change we now generate one package for each EFL module, just
as it was done in the 1.7.x series when we had different repositories.
This keeps the compatibility of the existing packages and removes the
need of specifying "Provides" and "Obsoletes" tags.

There is one additional package, efl-data, which ships the localization
files and Efl_Config.h. This file contains only information about the
version.

Change-Id: Iab11c00d07663159d18af24035c8fb7ad6be23dd
Signed-off-by: Philippe Coval <philippe.coval@open.eurogiciel.org>
Signed-off-by: Eduardo Lima (Etrunko) <eduardo.lima@intel.com>
---
 configure.ac | 4 ----
 efl.manifest | 6 ------
 2 files changed, 10 deletions(-)
 delete mode 100644 efl.manifest

diff --git a/configure.ac b/configure.ac
index 99b4288..98ef1d7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -612,10 +612,6 @@ EFL_SELECT_WINDOWS_VERSION
 
 ### Checks for libraries
 
-if test "${want_evas_engine_wayland_egl}" = "yes"; then
-   CFLAGS="$CFLAGS -I/usr/include/freetype2 -Ifribidi"
-fi
-
 EVIL_DLFCN_LIBS=""
 if test "${have_wince}" = "xyes"; then
    EFL_ADD_LIBS([EVIL], [-lws2])
diff --git a/efl.manifest b/efl.manifest
deleted file mode 100644
index 81ace0c..0000000
--- a/efl.manifest
+++ /dev/null
@@ -1,6 +0,0 @@
-<manifest>
-	<request>
-		<domain name="_"/>
-	</request>
-</manifest>
-
